snapcraft/plugins/copy.py

```diff
@@ -29,28 +44,37 @@ class CopyPlugin(snapcraft.BasePlugin):
 
     @classmethod
     def schema(cls):
-        return {
-            'properties': {
-                'files': {
-                    'type': 'object',
-                },
-            },
-            'required': [
-                'files',
-            ]
+        schema = super().schema()
+
+        schema['properties']['files'] = {
+            'type': 'object',
         }
 
+        # `source` key is not required, but the `files` key is.
```

Comment:
Nice, this is what I was coming into the review with my "tacones de punta" to see if you did :wink: